src/components/Technologies/CategorySelector/CloudProviderSelecor.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { Select } from "antd";
+import { cloudProviderTypes } from "../PackConstants";
+import styles from "./CategorySelector.module.scss";
+
+interface CloudProviderSelectorProps {
+  selected: string;
+  selectCloudProvider: (provider: string) => void;
+}
+
+export default function CloudProviderSelecor({ selected="", selectCloudProvider  }: CloudProviderSelectorProps) {
+  const getOptions = () => {
+    const providers = Object.keys(cloudProviderTypes);
```

Comment:
I think this should be an array instead. `Object.keys` order is not consistent in-between browsers.